GsaGH/Components/0_Model/SaveGsaModel.cs

```diff
@@ -105,16 +105,16 @@ public class SaveGsaModel : GH_OasysDropDownComponent {
     }
 
     internal void Save(ref GsaModel model, string fileNameAndPath) {
-      if (!fileNameAndPath.EndsWith(".gwb")) {
-        fileNameAndPath += ".gwb";
+      if (!fileNameAndPath.EndsWith(".gwa") && !fileNameAndPath.EndsWith(".gwa")) {
```

Comment:
should be gwb in one case?!

Reply:
yes, because we are checking if they NOT finishing with gwa or gwb